templates/application-insights.json

```diff
@@ -12,13 +12,6 @@
       "defaultValue": "",
       "type": "string"
     },
-    "availabilityTests": {
```

Comment:
Can we keep this availability test in here as well as the new file? All my templates are using this atm.

Reply:
Sure, didn't realise you were already using my availability test templates. Give me five to check in the code to restore it.